pom.xml

```diff
@@ -1,148 +1,158 @@
+        <artifactId>spark-bigquery-parent</artifactId>
+        <version>0.24.0-SNAPSHOT</version>
+        <relativePath>spark-bigquery-parent</relativePath>
+    </parent>
 
-  <artifactId>spark-bigquery-reactor</artifactId>
-  <packaging>pom</packaging>
-  <name>Spark BigQuery Connector Reactor</name>
+    <artifactId>spark-bigquery-reactor</artifactId>
+    <packaging>pom</packaging>
+    <name>Spark BigQuery Connector Reactor</name>
 
-  <licenses>
-    <license>
-      <name>Apache License, Version 2.0</name>
-      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
-      <distribution>repo</distribution>
-    </license>
-  </licenses>
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
 
-  <developers>
-    <developer>
-      <organization>Google Inc.</organization>
-      <organizationUrl>http://www.google.com</organizationUrl>
-    </developer>
-  </developers>
+    <developers>
+        <developer>
+            <organization>Google Inc.</organization>
+            <organizationUrl>http://www.google.com</organizationUrl>
+        </developer>
+    </developers>
 
-  <scm>
-    <connection>
-      scm:git:git@github.com:GoogleCloudDataproc/spark-bigquery-connector.git
-    </connection>
-    <developerConnection>
-      scm:git:git@github.com:GoogleCloudDataproc/spark-bigquery-connector.git
-    </developerConnection>
-    <url>git@github.com:GoogleCloudDataproc/spark-bigquery-connector.git</url>
-  </scm>
+    <scm>
+        <connection>
+            scm:git:git@github.com:GoogleCloudDataproc/spark-bigquery-connector.git
+        </connection>
+        <developerConnection>
+            scm:git:git@github.com:GoogleCloudDataproc/spark-bigquery-connector.git
+        </developerConnection>
+        <url>git@github.com:GoogleCloudDataproc/spark-bigquery-connector.git</url>
+    </scm>
 
-  <issueManagement>
-    <system>GitHub Issues</system>
-    <url>
-      https://github.com/GoogleCloudDataproc/spark-bigquery-connector/issues
-    </url>
-  </issueManagement>
+    <issueManagement>
+        <system>GitHub Issues</system>
+        <url>
+            https://github.com/GoogleCloudDataproc/spark-bigquery-connector/issues
+        </url>
+    </issueManagement>
 
-  <modules>
-    <module>spark-bigquery-parent</module>
-    <module>bigquery-connector-common</module>
-    <module>spark-bigquery-tests</module>
-    <module>spark-bigquery-connector-common</module>
-    <module>spark-bigquery-python-lib</module>
-    <module>spark-bigquery-pushdown</module>
-  </modules>
+    <modules>
+        <module>spark-bigquery-parent</module>
+        <module>bigquery-connector-common</module>
+        <module>spark-bigquery-tests</module>
+        <module>spark-bigquery-connector-common</module>
+        <module>spark-bigquery-python-lib</module>
```

Comment:
Do we not need `spark-bigquery-pushdown` here?

Reply:
No, it is not used in v2 at the moment, and there's no point to compile both 2.11 and 2.12 every time